src/tests/mock/tokens.js

```diff
@@ -0,0 +1,8 @@
+export const blaiseen =  "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImJsYWlzZWVuQGdtYWlsLmNvbSIsInByZWZlcmVkTGFuZyI6ImVuIiwiaWF0IjoxNTg1MzYxMzcwfQ.fXrqm6CGlturVlSAuGStJ6pZTE0tKA0lcLAaDMyCJHA"
+export const abdoulniyigena = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImFiZG91bG5peWlnZW5hQGdtYWlsLmNvbSIsInByZWZlcmVkTGFuZyI6ImZyIiwiaWF0IjoxNTg1MzYxMzQxfQ.ygfV3u-X2ZYm_LIqC6aLUbDz1MqPovlgH7KImytdDUg"
+export const blaisefr = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImJsYWlzZWZyQGdtYWlsLmNvbSIsInByZWZlcmVkTGFuZyI6ImVuIiwiaWF0IjoxNTg1MzYxMzAzfQ.qpuLVBXW21khrY_ZYX0-wqT6Xviy76TSi5uoKIqMup0"
+export const octopusbn = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6Im9jdG9wdXNibkBnbWFpbC5jb20iLCJwcmVmZXJlZExhbmciOiJlbiIsImlhdCI6MTU4NTM2MTI0M30.7R1fEtcHu-LJI9WseFsClQWV4nIk-MCGl_y07-dM9D0"
+export const testManager = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6Im5lZWRzLmdyaWRAZ21haWwuY29tIiwicHJlZmVyZWRMYW5nIjoiZW4iLCJpYXQiOjE1ODUzNjEyNzV9.Fxa-VmiWy8IieuAJTxQT3knApZPrhc_JVHWXrtGTctg"
+export const rusimbipatrick = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6InJ1c2ltYmlwYXRyaWNrQG91dGxvb2suY29tIiwicHJlZmVyZWRMYW5nIjoiZnIiLCJpYXQiOjE1ODUzNjEyMDV9.VBzPI5fnkDrfLisOmqxgXYUIid6m4r7QyAwNJM8xrKc"
+export const itsafact57 = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6Iml0c2FmYWN0NTdAZ21haWwuY29tIiwicHJlZmVyZWRMYW5nIjoiZnIiLCJpYXQiOjE1ODUzNjExNzF9.GvyrQOrq-7PgjYCuKgcfffABpvJqWz9dkMOArG565ZE"
+export const octopususer = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6Im9jdG9wdXN1c2VyQGFuZGVsYS5jb20iLCJwcmVmZXJlZExhbmciOm51bGwsImlhdCI6MTU4NTM2MTEyOX0.wcAWJWhp1SE1jnoKRrrTWPQ9tT-q7yzFLgyzIW-jYIM"
```

Comment:
please don't hard code tokens
just encode tokens using the function and the data you need in the token to test

Reply:
sure

Reply:
It slowed down the tests

Reply:
slowed down?